src/SWP/Bundle/MultiTenancyBundle/Tests/Command/CreateOrganizationCommandTest.php

```diff
@@ -0,0 +1,163 @@
+use SWP\Component\MultiTenancy\Repository\OrganizationRepositoryInterface;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class CreateOrganizationCommandTest extends \PHPUnit_Framework_TestCase
+{
+    private $commandTester;
+    private $command;
+    private $dialog;
+
+    public function setUp()
+    {
+        $application = new Application();
+        $application->add(new CreateOrganizationCommand());
+        $this->command = $application->get('swp:organization:create');
+        $this->dialog = $this->command->getHelper('dialog');
+    }
+
+    /**
+     * @covers SWP\Bundle\MultiTenancyBundle\Command\CreateOrganizationCommand
+     */
+    public function testExecuteWhenCreatingNewOrganization()
+    {
+        $this->dialog->setInputStream($this->getInputStream("Test\n"));
+        $this->command->setContainer($this->getMockContainer(null, new Organization(), 'Test'));
+        $this->commandTester = new CommandTester($this->command);
+        $this->commandTester->execute(['command' => $this->command->getName()]);
+
+        $this->assertRegExp(
+            '/Please enter name:Organization Test has been created and enabled!/',
+            $this->commandTester->getDisplay()
+        );
+    }
+
+    /**
+     * @covers SWP\Bundle\MultiTenancyBundle\Command\CreateOrganizationCommand
+     */
+    public function testExecuteWhenCreatingDefaultOrganization()
+    {
+        $this->command->setContainer($this->getMockContainer(null, new Organization()));
+        $this->commandTester = new CommandTester($this->command);
+
+        $this->commandTester->execute([
+            'command' => $this->command->getName(),
+            '--default' => true,
+        ]);
+
+        $this->assertRegExp(
+            '/Organization default has been created and enabled!/',
+            $this->commandTester->getDisplay()
+        );
+    }
+
+    /**
+     * @expectedException \InvalidArgumentException
+     * @covers SWP\Bundle\MultiTenancyBundle\Command\CreateOrganizationCommand
+     */
+    public function testExecuteWhenDefaultTenantExists()
+    {
+        $this->command->setContainer($this->getMockContainer(new Organization()));
+        $this->commandTester = new CommandTester($this->command);
+
+        $this->commandTester->execute([
+            'command' => $this->command->getName(),
+            '--default' => true,
+        ]);
+    }
+
+    /**
+     * @covers SWP\Bundle\MultiTenancyBundle\Command\CreateOrganizationCommand
+     */
+    public function testExecuteDisabledOrganization()
+    {
+        $this->dialog->setInputStream($this->getInputStream("Example\n"));
+        $this->command->setContainer($this->getMockContainer(null, new Organization(), 'Example'));
+        $this->commandTester = new CommandTester($this->command);
+        $this->commandTester->execute([
+            'command' => $this->command->getName(),
+            '--disabled' => true,
```

Comment:
It just the flag indicating which status should have a newly created organization, either it should be enabled or disabled by default. You can't disable/enable existing organizations so I think that a separate command is not needed. If you would want to disable/enable existing organizations then it makes sense to create `swp:organization:disable` and `swp:organization:enable` commands.